src/NLog/NewRelic.LogEnrichers.NLog/NewRelicJsonLayout.cs

```diff
@@ -18,6 +18,8 @@ public class NewRelicJsonLayout : JsonLayout
         private IJsonConverter _jsonConverter;
         private IJsonConverter JsonConverter => _jsonConverter ?? (_jsonConverter = ConfigurationItemFactory.Default.JsonConverter);
 
+        internal readonly static string UserPropertyKey = LoggingExtensions.UserPropertyPrefix.Substring(0, LoggingExtensions.UserPropertyPrefix.Length - 1);
```

Comment:
Perhaps use a TrimEnd('.') instead?  May read easier?

Reply:
Good idea, thanks for that suggestion.